src/app/(proper_react)/redesign/(authenticated)/user/dashboard/subscribed/page.tsx

```diff
@@ -54,7 +54,7 @@ export default async function Subscribed() {
       profileId,
       latestScan.scan.onerep_scan_id,
       scans,
-      "initial",
+      "manual",
```

Comment:
As I understand it (after having asked Rob about this a couple of days ago), `initial` is the status for the first scan that happens automatically after a profile has been activated - and once it's active, it'll periodically rescan with a `monitoring` status. `manual`, on the other hand, is for a one-time scan when the profile is not activated - i.e. for non-Premium customers.

I'm not sure if this code applies to a Premium or non-Premium customer. The error message referring to a manual scan above, together with the profile activation, is confusing.

Reply:
The issue here is that we can’t run an `initial` scan over and over again, because we decided to update the scans. One question I’d have for @rhelmer if we actually still need to run this everytime we visit `/subscribed` on dev.

Reply:
Ah, I got it now with the different scan types. I’ve changed how the scans are being updated — see: https://github.com/mozilla/blurts-server/pull/3470#discussion_r1338447632.